src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.Windows.crossruntime.cs

```diff
@@ -93,12 +93,14 @@ private static async Task<(ImageSource? Source, Color Background)> LoadSplashScr
 					return result;
 				}
 
+				// note: naive js parsing with string manipulation
+				// this may just fail, if we start to use nested object or escaped string containing '}'...
 				var startIdx = js!.IndexOf('{') + 1;
 				var endIdx = js.LastIndexOf('}') - 1;
-				var manifestProps = js.Substring(startIdx, endIdx - startIdx); // Trim "var UnoAppManifest = " from the start of the file so we're left with just the JSON
-				var manifest = manifestProps.Split(',')
-					.Select(x => x.Split(':', 2))
-					.ToDictionarySafe(x => x[0], x => x[1].Trim().Trim('\"'));
+				var manifestProps = js.Substring(startIdx, endIdx - startIdx); // Trim "var UnoAppManifest = " from the start of the file so we're left with just the inner of JSON
```

Comment:
@Xiaoy312 I am wondering if replacing manual string manipulation with JsonDocument would help having something more robust here?

Reply:
it could help, trade some perf for that safety... lets do that nexy
we are already in a try-catch from the caller,
catch it early or later doesn't change much here, the goal is to return successfully